packages/common-library/tests/test_json_serialization.py

```diff
@@ -95,3 +95,17 @@ def test_compatiblity_with_json_interface(
 
     # NOTE: cannot compare dumps directly because orjson compacts it more
     assert json_loads(orjson_dump) == json_loads(json_dump)
+
+
+def test_serialized_model_with_urls(faker: Faker):
```

Comment:
Nice! Just as a reference, add a link to this PR https://github.com/ITISFoundation/osparc-simcore/pull/6852